main.pm

```diff
@@ -857,6 +857,14 @@ else {
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
         loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ISCSI_SERVER")) {
+            set_var('INSTALLONLY', 1);
```

Comment:
Hmm, it did work and it is also in vars.json

Tumbleweed:
server: http://10.100.98.90/tests/1174
client: http://10.100.98.90/tests/1175